pkgs/development/tools/wally-cli/default.nix

```
@@ -1,28 +1,28 @@
-{ lib, buildGoPackage, fetchFromGitHub, pkg-config, libusb1 }:
+{ lib, buildGoModule, fetchFromGitHub, pkg-config, libusb1 }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "wally-cli";
-  version = "1.1.1";
+  version = "2.0.0-linux";
```

I find it a bit weird to have the platform in the version, but I see that they do that to their tags.
```suggestion
  version = "2.0.0";
```